_sass/tiny/_syntax.scss

```diff
@@ -0,0 +1,60 @@
+.highlight  { background: #ffffff; }
+.highlight .c { color: #999988; font-style: italic } /* Comment */
+.highlight .err { color: #a61717; background-color: #e3d2d2 } /* Error */
+.highlight .k { font-weight: bold } /* Keyword */
+.highlight .o { font-weight: bold } /* Operator */
```

Comment:
Use `//` comments everywhere<br>Merge rule `.highlight .o` with rule on line 1<br>Declaration should be terminated by a semicolon